react/SelectBox/SelectBox.jsx

```diff
@@ -19,7 +19,7 @@ const heights = {
 const customStyles = props => ({
   control: (base, state) => ({
     ...base,
-    backgroundColor: 'transparent',
+    backgroundColor: props.disabled ? 'transparent' : 'white',
```

Comment:
We want a transparent background when the selectbox is disabled ? 🤔

Reply:
Yes because the gray background color is managed via the `--disabled` class applied below. I am adding a comment.